ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/RestApiTest.java

```diff
@@ -499,6 +497,42 @@ public void shouldExecutePushQueryThatReturnsStreamOverRestV1() {
     assertThat(messages.get(3), is("{\"finalMessage\":\"Limit Reached\"}]"));
   }
 
+  @Test
+  public void shouldExecutePullQueryThatReturnsStreamOverRestV1AndIgnoreReset() {
+    // When:
+    final KsqlRequest request =
+        new KsqlRequest(
+            "SELECT USERID, PAGEID, VIEWTIME from " + PAGE_VIEW_STREAM + ";",
+            ImmutableMap.of("auto.offset.reset", "lastest"), // should get ignored
```

Comment:
Hey @vpapavas , I did also verify I got the same exception with exactly the config you reported (`earliest`). I figured I could make this test run double-duty as a regression test for the problem you reported and as a test that we really override the property, since setting the config to latest should _not_ return the rows below. So we are also verifying that the override in the code works properly.